e2e/tests/adminManagesOrders_test.js

```diff
@@ -81,12 +83,15 @@ Scenario('Create 32b discharge of care order', async ({ I, caseViewPage, manageO
   assertOrder(I, caseViewPage, {
     orderIndex: 1,
     orderType: 'Discharge of care order (C32B)',
-    approvalDate: today,
+    approvalDate: approvalDate,
   });
 });
 
 Scenario('Create EPO order', async ({ I, caseViewPage, manageOrdersEventPage }) => {
-  await setupScenario(I, caseViewPage);
+  const newCaseId = await I.submitNewCaseWithData(caseData);
```

Comment:
What is the reason for submitting a new case here?

Reply:
For both EPO had to keep the same dates, as this is constraint to the present and 8 days margin, so I can't use the sequence. In this way I put it in their own case, so don't need too much magic